_includes/instrument-mode.md

```diff
@@ -0,0 +1,117 @@
+{% include api_rersource = include.api_resouce %}
+{% include documentation_section =include.documentation_section %}
```

Comment:
The correct syntax here seems to be `assign` and not `include`, but we've experienced that the best is to not `assign` the `include` parameters, but instead refer directly to i.e. `include.api_resouce` everywhere. So the best would be to simply remove these two lines and change the references from `api_resource` to `include.api_resource`.

```suggestion
```